docs/security/trust/content_trust.md

```diff
@@ -0,0 +1,262 @@
+<!--[metadata]>
++++
+title = "Content trust in Docker"
+description = "Enabling content trust in Docker"
+keywords = ["trust, security, docker,  documentation"]
+[menu.main]
+parent= "smn_content_trust"
+weight=-1
++++
+<![end-metadata]-->
+
+# Content trust in Docker
+
+When transferring data among networked systems, *trust* is a central concern.
+When using Docker you push and pull images (data) to a registry. Currently,
+trust is supported for users of Docker's public Hub. It is not available for
+Docker Trusted Registry or for private registries. 
+
+## Understand trust in Docker 
+
+Content trust allows operations with a remote Docker registry to enforce
+client-side signing and verification of image tags. Currently, content trust is
+disabled by default. You must enabled it by setting the `DOCKER_CONTENT_TRUST`
+variable.
+
+Once trust is enabled, image creators can sign their images. Image consumers can
+ensure that the images they use are signed. Creators and consumers can be
+individuals alone or in organizations. Docker's trust feature supports users and
```

Comment:
`Docker's content trust supports...` (remove "feature" and always use "content trust")